### ISSUE TITLE
Not compatible with F3DEX2?

### ISSUE THREAD

**Issue by NintendoManiac64** [DavidSM64/SM64-Fast3D-AA-Reducer#2]
SM64 Lug's Delightful Dioramas on Console:

- https://www.romhacking.net/hacks/6427

This real-hardware patch for an existing SM64 mod implemented the F3DEX2 engine (rather than the original F3DEX engine) in order to improve performance, but this additionally means that your AA-Reducer utility is not compatible which I was hoping to be able to use to farther improve performance.

**Comment by DavidSM64** [comment]
Nope, sorry. This tool only supports the original Fast3D microcode. Adding F3DEX2 support would take a lot of work, since I know very little about how that microcode works.
